src/config.ts

```diff
@@ -37,15 +37,19 @@ async function set_up_configuration_file_or_url(
     optional = false
 ): Promise<void> {
     const source: string = core.getInput(name)
+
     if (source === '' && optional) {
         // TODO check that ort understands empty curations.yml file
-        await writeFile(target_filename, '')
+        await fs.promises.writeFile(target_filename, '')
+        return
```

Comment:
I needed a return here otherwise the github action fails. not sure why, prolly something to do with the asyncness